modules/dnn/perf/perf_net.cpp

```diff
@@ -155,7 +157,10 @@ PERF_TEST_P_(DNNTestNetwork, Inception_v2_SSD_TensorFlow)
 
```

Comment:
Lost changes for `Inception_v2_SSD_TensorFlow`:

https://github.com/opencv/opencv/blame/4.9.0/modules/dnn/perf/perf_net.cpp#L216